lib/cog/commands/bundle.ex

```diff
@@ -19,14 +19,10 @@ defmodule Cog.Commands.Bundle do
     "info <bundle>" => "Detailed information on an installed bundle",
     "enable <bundle> [<version>]" => "Enable a specific version of an installed bundle",
     "disable <bundle>" => "Disable an installed bundle",
-    "versions <bundle>" => "List all installed versions for a given bundle"
+    "versions <bundle>" => "List all installed versions for a given bundle",
+    "install <bundle>[:<version>]" => "Install latest or specified version of bundle from registry"
```

Comment:
I thought we went with version being an optional standalone argument, instead of tacked on with a colon?